tests/rustdoc/async-fn-opaque-item.rs

```diff
@@ -0,0 +1,14 @@
+// compile-flags: --document-private-items --crate-type=lib
```

Comment:
Oh btw, this line is useless.

Reply:
Do we always document private items?

Reply:
Oh, you mean the opaque types was private? Sorry, completely missed this part.